Only in src/ya_market: .DS_Store
Binary files target/ya_market/test/__pycache__/test_agreement.cpython-36-pytest-5.4.2.pyc and src/ya_market/test/__pycache__/test_agreement.cpython-36-pytest-5.4.2.pyc differ
Binary files target/ya_market/test/__pycache__/test_agreement_event.cpython-36-pytest-5.4.2.pyc and src/ya_market/test/__pycache__/test_agreement_event.cpython-36-pytest-5.4.2.pyc differ
Binary files target/ya_market/test/__pycache__/test_agreement_event_all_of.cpython-36-pytest-5.4.2.pyc and src/ya_market/test/__pycache__/test_agreement_event_all_of.cpython-36-pytest-5.4.2.pyc differ
Binary files target/ya_market/test/__pycache__/test_demand.cpython-36-pytest-5.4.2.pyc and src/ya_market/test/__pycache__/test_demand.cpython-36-pytest-5.4.2.pyc differ
Binary files target/ya_market/test/__pycache__/test_demand_offer_base.cpython-36-pytest-5.4.2.pyc and src/ya_market/test/__pycache__/test_demand_offer_base.cpython-36-pytest-5.4.2.pyc differ
Binary files target/ya_market/test/__pycache__/test_offer.cpython-36-pytest-5.4.2.pyc and src/ya_market/test/__pycache__/test_offer.cpython-36-pytest-5.4.2.pyc differ
Binary files target/ya_market/test/__pycache__/test_proposal.cpython-36-pytest-5.4.2.pyc and src/ya_market/test/__pycache__/test_proposal.cpython-36-pytest-5.4.2.pyc differ
diff -r -c target/ya_market/test/test_agreement.py src/ya_market/test/test_agreement.py
*** target/ya_market/test/test_agreement.py	2020-07-21 19:38:43.000000000 +0200
--- src/ya_market/test/test_agreement.py	2020-07-21 18:58:00.000000000 +0200
***************
*** 14,19 ****
--- 14,21 ----
  
  import ya_market
  from ya_market.models.agreement import Agreement  # noqa: E501
+ from ya_market.models.demand import Demand
+ from ya_market.models.offer import Offer
  from ya_market.rest import ApiException
  
  
***************
*** 35,42 ****
          if include_optional:
              return Agreement(
                  agreement_id="0",
!                 demand=None,
!                 offer=None,
                  valid_to=datetime.datetime.strptime(
                      "2013-10-20 19:20:30.00", "%Y-%m-%d %H:%M:%S.%f"
                  ),
--- 37,48 ----
          if include_optional:
              return Agreement(
                  agreement_id="0",
!                 demand=Demand(
!                     properties={}, constraints="0", demand_id="0", requestor_id="0"
!                 ),
!                 offer=Offer(
!                     properties={}, constraints="0", offer_id="0", provider_id="0"
!                 ),
                  valid_to=datetime.datetime.strptime(
                      "2013-10-20 19:20:30.00", "%Y-%m-%d %H:%M:%S.%f"
                  ),
***************
*** 51,58 ****
          else:
              return Agreement(
                  agreement_id="0",
!                 demand=None,
!                 offer=None,
                  valid_to=datetime.datetime.strptime(
                      "2013-10-20 19:20:30.00", "%Y-%m-%d %H:%M:%S.%f"
                  ),
--- 57,68 ----
          else:
              return Agreement(
                  agreement_id="0",
!                 demand=Demand(
!                     properties={}, constraints="0", demand_id="0", requestor_id="0"
!                 ),
!                 offer=Offer(
!                     properties={}, constraints="0", offer_id="0", provider_id="0"
!                 ),
                  valid_to=datetime.datetime.strptime(
                      "2013-10-20 19:20:30.00", "%Y-%m-%d %H:%M:%S.%f"
                  ),
diff -r -c target/ya_market/test/test_agreement_event.py src/ya_market/test/test_agreement_event.py
*** target/ya_market/test/test_agreement_event.py	2020-07-21 19:38:43.000000000 +0200
--- src/ya_market/test/test_agreement_event.py	2020-07-21 18:56:23.000000000 +0200
***************
*** 14,19 ****
--- 14,21 ----
  
  import ya_market
  from ya_market.models.agreement_event import AgreementEvent  # noqa: E501
+ from ya_market.models.demand import Demand
+ from ya_market.models.offer import Offer
  from ya_market.rest import ApiException
  
  
***************
*** 36,43 ****
              return AgreementEvent(
                  agreement=ya_market.models.agreement.Agreement(
                      agreement_id="0",
!                     demand=None,
!                     offer=None,
                      valid_to=datetime.datetime.strptime(
                          "2013-10-20 19:20:30.00", "%Y-%m-%d %H:%M:%S.%f"
                      ),
--- 38,49 ----
              return AgreementEvent(
                  agreement=ya_market.models.agreement.Agreement(
                      agreement_id="0",
!                     demand=Demand(
!                         properties={}, constraints="0", demand_id="0", requestor_id="0"
!                     ),
!                     offer=Offer(
!                         properties={}, constraints="0", offer_id="0", provider_id="0"
!                     ),
                      valid_to=datetime.datetime.strptime(
                          "2013-10-20 19:20:30.00", "%Y-%m-%d %H:%M:%S.%f"
                      ),
diff -r -c target/ya_market/test/test_agreement_event_all_of.py src/ya_market/test/test_agreement_event_all_of.py
*** target/ya_market/test/test_agreement_event_all_of.py	2020-07-21 19:38:43.000000000 +0200
--- src/ya_market/test/test_agreement_event_all_of.py	2020-07-21 18:55:48.000000000 +0200
***************
*** 14,19 ****
--- 14,21 ----
  
  import ya_market
  from ya_market.models.agreement_event_all_of import AgreementEventAllOf  # noqa: E501
+ from ya_market.models.demand import Demand
+ from ya_market.models.offer import Offer
  from ya_market.rest import ApiException
  
  
***************
*** 36,43 ****
              return AgreementEventAllOf(
                  agreement=ya_market.models.agreement.Agreement(
                      agreement_id="0",
!                     demand=None,
!                     offer=None,
                      valid_to=datetime.datetime.strptime(
                          "2013-10-20 19:20:30.00", "%Y-%m-%d %H:%M:%S.%f"
                      ),
--- 38,49 ----
              return AgreementEventAllOf(
                  agreement=ya_market.models.agreement.Agreement(
                      agreement_id="0",
!                     demand=Demand(
!                         properties={}, constraints="0", demand_id="0", requestor_id="0"
!                     ),
!                     offer=Offer(
!                         properties={}, constraints="0", offer_id="0", provider_id="0"
!                     ),
                      valid_to=datetime.datetime.strptime(
                          "2013-10-20 19:20:30.00", "%Y-%m-%d %H:%M:%S.%f"
                      ),
diff -r -c target/ya_market/test/test_demand.py src/ya_market/test/test_demand.py
*** target/ya_market/test/test_demand.py	2020-07-21 19:38:43.000000000 +0200
--- src/ya_market/test/test_demand.py	2020-07-21 18:47:34.000000000 +0200
***************
*** 35,44 ****
          # model = ya_market.models.demand.Demand()  # noqa: E501
          if include_optional:
              return Demand(
!                 properties=None, constraints="0", demand_id="0", requestor_id="0"
              )
          else:
!             return Demand(properties=None, constraints="0",)
  
      def testDemand(self):
          """Test Demand"""
--- 35,44 ----
          # model = ya_market.models.demand.Demand()  # noqa: E501
          if include_optional:
              return Demand(
!                 properties={}, constraints="0", demand_id="0", requestor_id="0"
              )
          else:
!             return Demand(properties={}, constraints="0",)
  
      def testDemand(self):
          """Test Demand"""
diff -r -c target/ya_market/test/test_demand_offer_base.py src/ya_market/test/test_demand_offer_base.py
*** target/ya_market/test/test_demand_offer_base.py	2020-07-21 19:38:43.000000000 +0200
--- src/ya_market/test/test_demand_offer_base.py	2020-07-21 18:47:18.000000000 +0200
***************
*** 33,41 ****
              optional params are included """
          # model = ya_market.models.demand_offer_base.DemandOfferBase()  # noqa: E501
          if include_optional:
!             return DemandOfferBase(properties=None, constraints="0")
          else:
!             return DemandOfferBase(properties=None, constraints="0",)
  
      def testDemandOfferBase(self):
          """Test DemandOfferBase"""
--- 33,41 ----
              optional params are included """
          # model = ya_market.models.demand_offer_base.DemandOfferBase()  # noqa: E501
          if include_optional:
!             return DemandOfferBase(properties={}, constraints="0")
          else:
!             return DemandOfferBase(properties={}, constraints="0",)
  
      def testDemandOfferBase(self):
          """Test DemandOfferBase"""
diff -r -c target/ya_market/test/test_offer.py src/ya_market/test/test_offer.py
*** target/ya_market/test/test_offer.py	2020-07-21 19:38:43.000000000 +0200
--- src/ya_market/test/test_offer.py	2020-07-21 18:46:54.000000000 +0200
***************
*** 35,44 ****
          # model = ya_market.models.offer.Offer()  # noqa: E501
          if include_optional:
              return Offer(
!                 properties=None, constraints="0", offer_id="0", provider_id="0"
              )
          else:
!             return Offer(properties=None, constraints="0",)
  
      def testOffer(self):
          """Test Offer"""
--- 35,44 ----
          # model = ya_market.models.offer.Offer()  # noqa: E501
          if include_optional:
              return Offer(
!                 properties={}, constraints="0", offer_id="0", provider_id="0"
              )
          else:
!             return Offer(properties={}, constraints="0",)
  
      def testOffer(self):
          """Test Offer"""
diff -r -c target/ya_market/test/test_proposal.py src/ya_market/test/test_proposal.py
*** target/ya_market/test/test_proposal.py	2020-07-21 19:38:43.000000000 +0200
--- src/ya_market/test/test_proposal.py	2020-07-21 18:46:23.000000000 +0200
***************
*** 35,41 ****
          # model = ya_market.models.proposal.Proposal()  # noqa: E501
          if include_optional:
              return Proposal(
!                 properties=None,
                  constraints="0",
                  proposal_id="0",
                  issuer_id="0",
--- 35,41 ----
          # model = ya_market.models.proposal.Proposal()  # noqa: E501
          if include_optional:
              return Proposal(
!                 properties={},
                  constraints="0",
                  proposal_id="0",
                  issuer_id="0",
***************
*** 43,49 ****
                  prev_proposal_id="0",
              )
          else:
!             return Proposal(properties=None, constraints="0",)
  
      def testProposal(self):
          """Test Proposal"""
--- 43,49 ----
                  prev_proposal_id="0",
              )
          else:
!             return Proposal(properties={}, constraints="0",)
  
      def testProposal(self):
          """Test Proposal"""
